node.js/.eslintrc

```diff
@@ -0,0 +1,203 @@
+{
```

Comment:
Seems a little unusual to me to include a `.eslintrc` in a library.

Reply:
It's just for linting purposes

Reply:
Right... linting should probably be done externally, not included in a public library itself. Seems strange to me.

Reply:
`ESLint`ing is part of `Continuous Testing` inside `CircleCI`.